integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -156,8 +159,11 @@ func runMultipleRoundsOfTheGame(
 
 		// waiting to disseminate transactions
 		time.Sleep(stepDelay)
-
-		round, nonce = integrationTests.ProposeAndSyncBlocks(t, nodes, idxProposers, round, nonce)
+		integrationTests.ProposeBlock(nodes, idxProposers, round, nonce)
```

Comment:
isn't this the implementation of integrationTests.ProposeAndSyncBlocks? Duplicate with the code on L176-L180

Reply:
thought it was the error. reverted.